spec/features/pages/index_spec.rb

```diff
@@ -42,5 +42,17 @@
       expect(page).to have_button('Shuffle')
     end
   end
+
+  xit "When a visitor visits the home page can go to a card's show" do
```

Comment:
🚨 I need to find a way to look at the object so I can provide the id for the path  🚨